packages/@aws-cdk/aws-glue/README.md

```diff
@@ -204,4 +202,51 @@ new glue.Table(stack, 'MyTable', {
 |-------------------------------------	|----------	|-------------------------------------------------------------------	|
 | array(itemType: Type)               	| Function 	| An array of some other type                                       	|
 | map(keyType: Type, valueType: Type) 	| Function 	| A map of some primitive key type to any value type                	|
-| struct(collumns: Column[])          	| Function 	| Nested structure containing individually named and typed collumns 	|
+| struct(columns: Column[])          	| Function 	| Nested structure containing individually named and typed columns 	|
+
+### Job
+
+A `Job` is the business logic that performs the extract, transform, and load (ETL) work in AWS Glue. When you start a job, AWS Glue runs a script that extracts data from sources, transforms the data, and loads it into targets.
+
+The only required parameter is `script` which defines the relative path to your Python or Scala file. A `CustomResource` pushes your script to S3 and configures the `Job` with necessary permissions and the final location of the script.
+
+```ts
+new glue.Job(stack, 'MyJob', {
+  script: './lib/scripts/etlJob.py'
+});
+```
+
+By default, an S3 bucket will be created to store the provided script but you can manually pass the `bucket` and `s3Prefix`:
+
```

Comment:
I feel like it is more natural to use the s3-assets module here. This would work similar to how Lambda deals with assets.

Reply:
@adamelmore I agree with @tdikland that we should just use the regular asset mechanism. 

Regarding the API, I want to provide something similar to what we have in [`Lambda`](https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-lambda#handler-code) and [`CloudWatch Synthetics`](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-synthetics/README.md#canary), which also accept code from the user.

```ts
new glue.Job(this, 'Job', {
  code: glue.Code.fromAsset(...) // glue.Code.fromInline(...)
});

```

Is it even possible to use a directory or does it have to be a single script file?

Reply:
This all sounds good to me. And pretty certain only a single script is allowed.

Reply:
In that case the api shouldn't `Code.fromAsset` but rather `Code.fromScript` or `Code.fromFile`